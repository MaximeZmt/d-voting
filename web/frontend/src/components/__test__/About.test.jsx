import React from 'react';
import About from '../About';
import renderer from 'react-test-renderer';

describe('About', () => {
  it('should render the About Component correctly in English', () => {
    const component = renderer.create(<About />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('About component renders its text', () => {
    const component = renderer.create(<About />);
    expect(component.root.findByProps({ className: 'about-text' }).children);
  });
});
