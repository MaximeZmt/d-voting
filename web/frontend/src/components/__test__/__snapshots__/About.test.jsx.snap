// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render the About Component correctly in English 1`] = `
<div
  className="about-container"
>
  <div
    className="about-text"
  >
    <br />
    about1
    <br />
    <br />
    about2
    <br />
    <br />
    about3
    <br />
    <br />
    about4
    <br />
  </div>
</div>
`;
